### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Sboursen**
# Portfolio: develop desktop version using media queries

> The template used: [Template 3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)

## PR cover these tasks:

- Create a desktop version according to the aforementioned template.
- Use media queries to implement a responsive web page that toggles between desktop and mobile
versions at the `992px` mark.
- Make sure the desktop version _looks good_ in the range `992px - 1100px`.
- Make sure the web page _looks good_ in the transition between mobile and desktop versions.
- Use both _Flexbox_ and _CSS Grid_ for the layout.
- Stick the template as much as possible.

## Screenshots of the tasks achieved:

<details>
  <summary>Please click to view hello section</summary>

[Hello section desktop version](https://github.com/Sboursen/My-portfolio/blob/61948ae55272d50b73912ef0875bc33563c497bb/resources/app-screenshot-hello-section-desktop.png) 
</details>

<details>
  <summary>Please click to view works section</summary>

[Works section desktop version](https://github.com/Sboursen/My-portfolio/blob/61948ae55272d50b73912ef0875bc33563c497bb/resources/app-screenshot-hello-section-desktop.png) 
</details>

<details>
  <summary>Please click to view about me section</summary>

[About me section desktop version](https://github.com/Sboursen/My-portfolio/blob/61948ae55272d50b73912ef0875bc33563c497bb/resources/app-screenshot-about-me-section-desktop.png) 
</details>

<details>
  <summary>Please click to view contact me section</summary>

[Contact me section desktop version](https://github.com/Sboursen/My-portfolio/blob/61948ae55272d50b73912ef0875bc33563c497bb/resources/app-screenshot-contact-me-section-desktop.png) 
</details>

**Looking forward for your review**
**Best,**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -446,3 +480,136 @@ nav {
...
+  }
+
+  .about-me .about-me-card {
+    margin: 0;
+  }
+
+  .about-me .about-me-card:nth-child(2) {
+    margin-left: 142px;
+    grid-column: 1 / 3;
+  }
```
Comment (Meltrust) [inline style.css]: - From the "About me" section, please kindly vertically center those containers, like in the Figma template.

Your Implementation | Figma Design
:---:|:---:
image | image

That way, the project will look a lot more like the template! 💪

File: index.html [hunk 2]
```diff
@@ -206,41 +249,43 @@ <h3 class="card-title">Skills</h3>
     </section>
 
   </Main>
+
   <footer class="contact-form">
-    <div class="footer-text">
-      <h2 class="title">Contact Me</h2>
+    <h2 class="title">Contact Me</h2>
+    <div class="footer-content">
```
Comment (Meltrust) [inline index.html:252]: - [x] In the "I can help you build..." text, please kindly correct the font style by checking the template.
- [x] In the form placeholders, you can change the font style and family by using the pseudoclass `::placeholder`.  More info [here](https://www.w3schools.com/cssref/sel_placeholder.asp).
- [x] In the Figma template, the border for the `textarea` input is a different color (green).  Let's kindly correct it.
- [x] There is a horizontal line that should be over the icons, but it's missing 😢.  Please kindly add it.

Your Implementation | Figma Design
:---:|:---:
image | image

That way, this particularly beautiful project will be finished!! 💯

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Sboursen, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## This is one of the most beautiful projects I've reviewed in a long time.  I was really impressed and delighted.  👏   Just fix those little details I left for you and this will be approved really quickly!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Your project is beautiful.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Sboursen) [reply style.css]: Good catch. I didn't notice this. :thumbsup: :thumbsup:

File: index.html (hunk 2 above)
Comment (Sboursen) [reply index.html:252]: Thanks,
This is a nicely detailed review.
I'll make sure to make all the changes
:smile:

**Comment by Sboursen** [comment]
> Hi @Sboursen,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## This is one of the most beautiful projects I've reviewed in a long time. I was really impressed and delighted.  Just fix those little details I left for you and this will be approved really quickly!
> ## Required Changes 
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * Your project is beautiful.
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @Meltrust!
Thanks a lot for the kind words :blush: 
This is probably the best review I got detail-wise.

**Comment by Sboursen** [comment]
Hi!
I have made the changes requested.
Please find below the app screenshots for the result after the changes.

<details>
  <summary>Please click to view about me section</summary>

[About me section desktop version](https://github.com/Sboursen/My-portfolio/blob/1688690f186861c196db0bc69bf102d29052f457/resources/app-screenshot-about-me-section-desktop.png) 
</details>

<details>
  <summary>Please click to view contact me section</summary>

[Contact me section desktop version](https://github.com/Sboursen/My-portfolio/blob/1688690f186861c196db0bc69bf102d29052f457/resources/app-screenshot-contact-me-section-desktop.png) 
</details>

**Review by Meltrust — APPROVED** [review]
Hi @Sboursen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Beautiful project ✔️ 
- Nice readme ✔️ 
- Nice PR comments. ✔️ 
- Corrected all the requirements. ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sboursen** [comment]
Thank you @Meltrust!
Hope to work with you in the future :smile:

**Pull request merged**